dockerfiles/Dockerfile.apisix.deb

```diff
@@ -0,0 +1,67 @@
+ARG image_base="ubuntu"
+ARG image_tag="focal"
+ARG iteration="0"
+ARG apisix_repo="https://github.com/apache/apisix"
+ARG checkout_v
+
+FROM ${image_base}:${image_tag}
+
+RUN set -x \
```

Comment:
Better to refactor this common and turn it into a script, instead of repeating it